### PR TITLE
added lesson_3 homework

### PR DESCRIPTION
**Pull request by Domerungus**

### CODE REVIEW

File: src/main/java/students/jegors_rutkovskis/lesson_3/level_7/BankAccount.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        this.money = money;
+    }
+
+    String getOwner() {
+        return this.owner;
+    }
+
+    String getMoney() {
+        return "" + money;
+    }
```
Comment (antdolg1) [inline src/main/java/students/jegors_rutkovskis/lesson_3/level_7/BankAccount.java:18]: You have variable money which is int, why you decided to convert it to `String` in such a way and not to return `int` value? According to usage in BankAccountDemo there is no need in such to be honest strange operation. Please change return type of `getMoney()` method to `int`.

File: src/main/java/students/jegors_rutkovskis/lesson_3/level_7/task_29/Car.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
+package students.jegors_rutkovskis.lesson_3.level_7.task_29;
+
+class Car {
+    private String carBrand;
+    private String carColor;
+    private int fuelTank;
+    private final int MAX_FUEL_IN_TANK = 20;
```
Comment (antdolg1) [inline src/main/java/students/jegors_rutkovskis/lesson_3/level_7/task_29/Car.java:7]: If you want to define constant, it is recommended to make it static as well, so it use less memory as value would be shared across all instances of the class (`private static final int MAX_FUEL_IN_TANK = 20`)

File: src/main/java/students/jegors_rutkovskis/lesson_3/level_7/task_29/Car.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    void accelerate(int speed){
+        if (currentSpeed + speed <= maxSpeed){
+            currentSpeed += speed;
+            System.out.println(currentSpeed());
+        }
+        else {
+            System.err.println("it is impossible to accelerate " +
+                    "because the maximum speed has been reached");
+        }
+        fuelTank -= 5;
```
Comment (antdolg1) [inline src/main/java/students/jegors_rutkovskis/lesson_3/level_7/task_29/Car.java:27]: Avoid such mystic numbers. It is better to provide a comment, on why we subtract 5 or create a separate variable for this number giving it a name, which would describe its purpose.

File: src/main/java/students/jegors_rutkovskis/lesson_3/level_7/task_29/Car.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
+package students.jegors_rutkovskis.lesson_3.level_7.task_29;
+
+class Car {
+    private String carBrand;
```
Comment (antdolg1) [inline src/main/java/students/jegors_rutkovskis/lesson_3/level_7/task_29/Car.java:4]: If you add `private` access modifier to class fields, then add getters/setters in order to interact with those values.

File: src/main/java/students/jegors_rutkovskis/lesson_3/level_7/task_29/Car.java [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+                    "because the maximum speed has been reached");
+        }
+        fuelTank -= 5;
+    }
+    void brake(){
+        currentSpeed = 0;
+        System.out.println("Car is stopped...");
+    }
+    private String currentSpeed(){
+        return "Current speed is "
```
Comment (antdolg1) [inline src/main/java/students/jegors_rutkovskis/lesson_3/level_7/task_29/Car.java:34]: As you declared your class fields private, please add getters/setters. Such approach as you used here and in other places violates principle of encapsulation. So add getters/setters and when you need to get value use getter, when you need to set value use setter.

**Review by antdolg1 — CHANGES_REQUESTED** [review]
It's cool that you are using some functions/classes which we did not talked about during a lecture, I like it but there are some things which needs to be improved, please see my comments.

P.S. Please do not close other students Pull Requests.

File: src/main/java/students/jegors_rutkovskis/lesson_3/level_7/BankAccount.java (hunk 1 above)
Comment (Domerungus) [reply src/main/java/students/jegors_rutkovskis/lesson_3/level_7/BankAccount.java:18]: Task 32 states that only the BankAccount class can be modified. 
String moneyAmount = bankAccount.getMoney();

**Review by antdolg1 — APPROVED** [review]
Good job!

**Pull request merged**